translator-files/po/es/Command-Line-Options.po

```diff
@@ -132,51 +130,24 @@ msgstr "### Configuraciones generales (modos cliente y servidor)"
-"| `-h,-?` | `--help`          | This help text                               "
-"                                                                |            "
-"                                              |\n"
-"| `-i`    | `--inifile`       | Set location of initialization file"
-" (overrides default)                                                      |"
-" Client (always) and server (with GUI)                    |\n"
-"| `-n`    | `--nogui`         | Disable GUI                                  "
-"                                                                |            "
-"                                              |\n"
-"| `-p`    | `--port`          | Local UDP port number. Default is 22124      "
-"                                                                |            "
-"                                              |\n"
-"|    `-Q` | `--qos`           | Sets the quality of service DSCP value."
-" Default is 128. Disable with 0.  | QoS is ignored by Windows, but see [Tips"
-" & Tricks](Tips-Tricks-More#quality-of-service) for how to apply it. |\n"
-"| `-t`    | `--notranslation` | Disable translations                         "
-"                                                                |            "
-"                                              |\n"
+"| Short   | Long name         | `[...]` means optional parameter                                                                             | Notes                                                    |\n"
+"|---------|-------------------|--------------------------------------------------------------------------------------------------------------|----------------------------------------------------------|\n"
+"| `-h,-?` | `--help`          | This help text                                                                                               |                                                          |\n"
+"| `-i`    | `--inifile`       | Set location of initialization file (overrides default)                                                      | Client (always) and server (with GUI)                    |\n"
+"| `-n`    | `--nogui`         | Disable GUI                                                                                                  |                                                          |\n"
+"| `-p`    | `--port`          | Local UDP port number. Default is 22124                                                                      |                                                          |\n"
+"|    `-Q` | `--qos`           | Sets the quality of service DSCP value. Default is 128. Disable with 0.  | QoS is ignored by Windows, but see [Tips & Tricks](Tips-Tricks-More#quality-of-service) for how to apply it. |\n"
+"| `-t`    | `--notranslation` | Disable translations                                                                                         |                                                          |\n"
```

Comment:
Needs cross checking in live site...

Reply:
Again, this is due to the text wrapping, no need to check. If you see that both the source and target segments have been rearranged but still remain as a validated pair, you know it's the editor just rearranging text. The output to the target .md file doesn't change.

When things settle down, what will probably happen is that each translator will consistently use their editor of choice for their language and these rearrangements won't happen so often. It's been mostly me up until now, and as soon as someone else uses a different editor (I don't like Qt Linguist at all), it happens.